Sources/NetworkProtection/Diagnostics/NetworkProtectionConnectionTester.swift

```diff
@@ -110,8 +100,7 @@ final class NetworkProtectionConnectionTester {
 
     deinit {
         os_log("[-] %{public}@", log: .networkProtectionMemoryLog, type: .debug, String(describing: self))
-
-        cancelTimerImmediately()
+        timer?.cancel()
```

Comment:
Since methods are now `@MainActor` it's not as easy to call `cancelImmediately`, but we just wanna make sure the timer is cancelled.